pkg/splunk/enterprise/names.go

```diff
@@ -306,3 +312,8 @@ var GetProbeMountDirectory = func() string {
 var GetProbeVolumePermission = func() int32 {
 	return probeVolumePermission
 }
+
+// GetLivenessDriverLocation returns the location of k8_liveness_driver.sh
+var GetLivenessDriverLocation = func() string {
+	return livenessDriverLocation
+}
```

Comment:
Probably don't need this since it is just returning a string. You can directly use `livenessDriverLocation`

Reply:
This gives us flexibility in unit test case to modify the path. Since it relative location it can cause issues when executing unit test.

Reply:
I agree with @pdhanoyasplunk , this simplifies writing unit test cases